pennylane/allocation.py

```diff
@@ -150,35 +150,27 @@ def deallocate(wires: DynamicWire | Wires | Sequence[DynamicWire]) -> Deallocate
 
     .. code-block:: python
 
-        @qml.qnode(qml.device('default.qubit'))
         def c():
             qml.H(0)
 
-            wires = qml.allocation.allocate(1, require_zeros=True)
+            wires = qml.allocation.allocate(1, require_zeros=True, restored=True)
             qml.CNOT((0, wires[0]))
             qml.CNOT((0, wires[0]))
-            qml.allocation.deallocate(wires, restored=True)
+            qml.allocation.deallocate(wires)
 
             new_wires = qml.allocation.allocate(1)
             qml.SWAP((0, new_wires[0]))
             qml.allocation.deallocate(new_wires)
 
-            return qml.probs(wires=0)
-
-
-    >>> print(qml.draw(c, level="user")())
+    >>> print(qml.draw(c)())
                 0: ──H────────╭●─╭●─────────────╭SWAP─────────────┤  Probs
```

Comment:
The rest should be fine. I'll approve once this change is merged.

```suggestion
                0: ──H────────╭●─╭●─────────────╭SWAP─────────────┤
```